addons/iap/models/iap_account.py

```diff
@@ -40,8 +40,8 @@ class IapAccount(models.Model):
     company_ids = fields.Many2many('res.company')
 
     # Dynamic fields, which are received from iap server and set when loading the view
-    balance = fields.Char(readonly=True)
-    balance_amount = fields.Float(compute='_compute_balance_amount')
+    balance_amount = fields.Float()
+    balance = fields.Char(compute='_compute_balance')
```

Comment:
Won't that require an upgrade script to remove the `balance` column from the table since it used to be stored before ?
Not sure, first time I encounter this case 🤔.

Reply:
Nevermind, I just saw you linked an upgrade PR that does just that :stuck_out_tongue: